bears/python/ShebangBear.py

```diff
@@ -0,0 +1,34 @@
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+            Ensure that the file begins with the Shebang operator.
+        """
+        if len(file) == 0 or '#!/usr/bin/env' in file[0]:
+            return
+
+        diff = Diff(file)
+        mes = ''
+
+        if '#!' in file[0]:
```

Comment:
@AMR-KELEG You can just grab the binary name (without the directory) and add that at the end of `/usr/bin/env `.

ex.
1. `#!/usr/bin/python` -> `#!/usr/bin/env python`
2. `#!/usr/bin/python3` -> `#!/usr/bin/env python3`
3. `#!/usr/bin/asuka --intepreter-mode --no-warnings` -> `#!/usr/bin/env asuka --intepreter-mode --no-warnings`